tmp/.grafbase/generated/schemas/schema.graphql

```diff
@@ -0,0 +1,15 @@
+extend schema
```

Comment:
Are these files supposed to be in the PR?

Reply:
nope, good catch